packages/gateway/src/index.ts

```diff
@@ -3,20 +3,27 @@ import { Command } from 'commander';
 import { readFileSync } from 'fs';
 import { ethers } from 'ethers';
 import { JSONDatabase } from './json';
-
-const program = new Command();
-program
-  .requiredOption(
-    '-k --private-key <key>',
-    'Private key to sign responses with. Prefix with @ to read from a file'
-  )
-  .requiredOption('-d --data <file>', 'JSON file to read data from')
-  .option('-t --ttl <number>', 'TTL for signatures', '300')
-  .option('-p --port <number>', 'Port number to serve on', '8000');
-
-program.parse(process.argv);
-
-const options = program.opts();
+import 'dotenv/config';
```

Comment:
What was wrong with using command-line options?

Reply:
I can't deploy to app engine with command line

Reply:
You can: https://cloud.google.com/appengine/docs/standard/nodejs/runtime#application_startup

Reply:
I don't understand, why can you not just leave this as-is for anyone running it using yarn, and include the arguments in app.yaml for the App Engine app?